sys/riscv/boot.c

```diff
@@ -59,23 +59,26 @@
  * will be dropped.
  */
 #define KL_LOG KL_INIT
+#include <sys/boot.h>
 #include <sys/fdt.h>
 #include <sys/kasan.h>
 #include <sys/klog.h>
+#include <sys/libkern.h>
 #include <sys/mimiker.h>
 #include <sys/pcpu.h>
 #include <sys/pmap.h>
 #include <riscv/abi.h>
 #include <riscv/cpufunc.h>
 #include <riscv/pmap.h>
+#include <libfdt/libfdt.h>
 
-#define PHYSADDR(x) ((paddr_t)((vaddr_t)(x) & ~KERNEL_VIRT) + KERNEL_PHYS)
+#define PHYSADDR(x) ((paddr_t)((vaddr_t)(x) + (KERNEL_PHYS - KERNEL_VIRT)))
```

Comment:
At the beginning of `sys/riscv/boot.c`, there's a macro called `RISCV_PHYSADDR`. Please change the macro as it's become inconsistent.

Reply:
Done!